changelog.txt

```diff
@@ -34,6 +34,9 @@ Removed a formal parameter from methods in `BaseTypeVisitor`:
 
 In Analysis.runAnalysisFor(), changed boolean parameter to enum BeforeOrAfter.
 
+Removed org.checkerframework.framework.util.AnnotatedTypes#getIteratedType; use
+AnnotatedTypeFactory#getIteratedType(ExpressionTree) instead.
```

Comment:
Should this be `#getIteratorElementType...`?

Reply:
Yes, thanks!